### PR TITLE
Added metrics

### PR DESCRIPTION
**Pull request by allenywang**
For issue #62

### CODE REVIEW

File: annotator/static/player.js [hunk 1]
```diff
@@ -193,6 +199,9 @@ class Player {
     }
 
     drawAnnotationOnRect(annotation, rect) {
+        if (this.metrics.annotationsStartTime == null) {
```
Comment (antingshen) [inline annotator/static/player.js:202]: should this (and the other time) be `== null` or `=== undefined`?

File: annotator/static/views/player.js [hunk 2]
```diff
@@ -239,18 +239,20 @@ class PlayerView {
...
-        if (currentTime < this.videoStart) {
-            closestTimeinRange = this.videoStart;
-        } else if (currentTime > this.videoEnd) {
-            closestTimeinRange = this.videoEnd;
-        }
-        return {
-            violatesStartTime: currentTime < this.videoStart,
-            violatesEndTime: currentTime > this.videoEnd,
-            closestTimeinRange: closestTimeinRange,
+        var currentTime =this.$('control-time').val();
```
Comment (antingshen) [inline annotator/static/views/player.js]: space. run linter :)

File: annotator/static/views/player.js [hunk 3]
```diff
@@ -332,6 +334,7 @@ class PlayerView {
         if (timeRange.violatesStartTime || timeRange.violatesEndTime) {
             this.fixVideoTime(timeRange.closestTimeinRange);
         }
+        console.log("T")
```
Comment (antingshen) [inline annotator/static/views/player.js]: Delete?

File: annotator/static/views/player.js [hunk 4]
```diff
@@ -346,7 +349,7 @@ class PlayerView {
         var timeRange = this.checkTimeRange();
         this.$('control-scrubber:not(:active)').val(value * this.CONTROL_SCRUBBER_GRANULARITY / this.video.duration);
         if (timeRange.violatesStartTime || timeRange.violatesEndTime) {
-            this.fixVideoTime();
+            this.fixVideoTime(timeRange.closestTimeinRange);
```
Comment (antingshen) [inline annotator/static/views/player.js]: This is still here

File: annotator/static/player.js (hunk 1 above)
Comment (allenywang) [inline annotator/static/player.js:202]: Earlier I set it to null on line 26.

Comment (antingshen) [inline annotator/static/player.js:202]: Then it should be `=== null`. `== null` means `null` or `undefined`

**Pull request closed without merging**